scripts/bashbrew-host-arch.sh

```diff
@@ -28,7 +30,8 @@ case "$arch" in
 	amd64 | x86_64)    found 'amd64'    ;;
 	arm64 | aarch64)   found 'arm64v8'  ;;
 	armel)             found 'arm32v5'  ;;
-	armv7)             found 'arm32v7'  ;;
+	armv6*)            found 'arm32v6'  ;;
```

Comment:
Technically, I could add `arm32v5` here too, but I don't think we want to encourage `arm32v5` users -- their time is just shy of completely up. :sweat_smile: